website/docs/reference/event-types.mdx

```diff
@@ -574,12 +574,6 @@ This event fires when you drop all existing tags as part of a configuration impo
 
 ### `feature-potentially-stale-on`
```

Comment:
do we not want to put event availibility in? this was released at some point, but not available to users previously. I wonder if that's useful info or not?